README.md

```diff
@@ -0,0 +1,101 @@
+Circle Image
+
+This library is developed to provide circle image implemented using extended typescript.
+
+Download and Install
+
+npm install @ohos/libcircularimage --save
+
+For more information on OpenHarmony npm environment configuration, please refer to How to install OpenHarmony npm package .
+
+Instructions for Use:
+
+1. Import files and code dependencies
+
+import {CircularImage} from '@ohos/libcirclularimage'
```

Comment:
import  { CircularImage, CircleImageModel, ClickListener } from '@ohos/libcircularimage'

change to include all of this